launch.go

```diff
@@ -78,13 +81,36 @@ func postLaunchHandler(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func quickLauncherHandler(w http.ResponseWriter, r *http.Request) {
+	log.Println("Request Received!")
+	hostURL := settings.Get("SURVEY_RUNNER_URL")
+	surveyURL := r.URL.Query().Get("url")
```

Comment:
We should probably validate that this is a valid URL before continuing to request it.